manifests/init.pp

```diff
@@ -114,10 +114,12 @@
   $prd_web_ui_logger_syslog_errors = true,
   $prd_web_ui_port                 = '3080',
   $prd_web_ui_timeout              = '300',
+  $prd_web_ui_base_url             = undef,
 
   ## JSON API Params
   $prd_json_api_access_log           = 'jsonapi_access.log',
   $prd_json_api_base_url             = 'http://localhost',
+  $prd_json_api_url                  = undef,
```

Comment:
In my point of view, the `$prd_json_api_base_url` is the internal url (behind a proxy). The `$prd_json_api_url` is the public url in front of the proxy.
Internal we run the flapjack-api on http://localhost:3081/ and external on https://hostname/flapjack/api/ (where apache is the proxy wo cares about the authentication)